app/models/card.rb

```diff
@@ -3,6 +3,15 @@ class Card < ActiveRecord::Base
   validates :original_text, :translated_text, :add_review_date, presence: true
   validate :words_cannot_be_equal
 
+  scope :random_for_review, -> { where('review_date <= ?', Time.now)
+                                   .order('RANDOM()') }
```

Comment:
Expression at 7, 55 should be on its own line.<br>Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 33) spaces for indenting an expression spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.